Update/Add Targets that Acquire AWX Version

beeankha opened the pull request:
Changing the Makefile so that:
1) Python 3 vs stricter 3.8 is used
2) Hiding stderr on `make VERSION` target
3) Creating a new target for Collections version since we can't build collections using version output in the format of something like `awx: 19.4.1.dev84+g00faf7a2f1`

shanemcd commented on Makefile:
```diff
@@ -1,12 +1,13 @@
-PYTHON ?= python3.8
```
This is most likely why CI is failing.

shanemcd commented on tools/ansible/roles/dockerfile/templates/Dockerfile.j2:
```diff
@@ -36,16 +36,16 @@ RUN dnf -y update && \
-    python38-pip \
-    python38-psycopg2 \
-    python38-setuptools \
+    python3-devel \
```
I'm pretty sure that this will resolve to python3.6 on centos 8, which we definitely dont want.

shanemcd reviewed (changes requested):
I think almost everything here needs to get reverted. 

But I also see the problem you were trying to solve, and can help you look for a more surgical fix.

shanemcd committed:
Fix `make install_collection`

The version obtained from setuptools_scm is not compatible with ansible-galaxy collection install.
